modules/base/rendering/pointcloud/renderableinterpolatedpoints.cpp

```diff
@@ -187,11 +187,11 @@ RenderableInterpolatedPoints::Interpolation::Interpolation()
         );
         // No syncing, as this was triggered from a property change (which happened
         // based on an already synced script)
-        global::scriptEngine->queueScript(
-            script,
-            scripting::ScriptEngine::ShouldBeSynchronized::No,
-            scripting::ScriptEngine::ShouldSendToRemote::No
-        );
+        global::scriptEngine->queueScript({
+            .code = script,
+            .synchronized = scripting::ScriptEngine::Script::ShouldBeSynchronized::No,
```

Comment:
eek, the path to this boolean type became very long now

Oh well

Reply:
It's partially on purpose to not make it easy to write this, since it could result in hard-to-detect errors and we want to encourage the use of the default values